_posts/2019-05-21-accessing-kafka-part-5.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+title:  "Accessing Kafka: Part 4 - Load Balancers"
+date: 2019-05-13
+author: jakub_scholz
+---
+
+In the fifth and last part of this blog post series we will look at exposing Kafka using Kubernetes Ingress.
+This post will explain how to use Ingress controllers on Kubernetes, how Ingress compares with OpenShift Routes and how it can be used with Strimzi and Kafka.
+Off-cluster access using Kubernetes Ingress is available only from Strimzi 0.12.0.
```

Comment:
I guess we cannot publish the article until this release will be out.

Reply:
I don't think so - it is just a preview. Also the other posts mentioned some un-released features.